addons/wiki/static/dragNDrop.js

```diff
@@ -68,6 +68,7 @@ var localFileHandler = function(files, cm, init, fixupInputArea, newName) {
                     editor.enable();
                 } else {
                     var waterbutlerURL = ctx.waterbutlerURL + 'v1/resources/' + ctx.node.id + '/providers/osfstorage' + encodeURI(path) + '?name=' + encodeURI(name) + '&type=file';
+                    $osf.trackClick('wiki-markdown-drop-image', ctx.node.id, 'dropped-image');
```

Comment:
@Johnetordoff, I'm not very familiar with our event tracking, but the signature for `trackClick` expects a category, action, and label. So in this case:
```
category = 'wiki-markdown-drop-image'
action = the node guid
label = 'dropped-image'
```
I'm wondering if it makes sense to make the category more general. Maybe just "wiki" in case additional event tracking is added to this page? I'd propose:
```
category = 'wiki-page'
action = 'wiki-edit'
label = 'dropped-image'
```
Thoughts? This is by no means a blocker.